frontend/src/components/TopBar/RealmSelect.tsx

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: 2024 German Aerospace Center (DLR) and CISPA Helmholtz Center for Information Security
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import {FormControl, InputLabel, Select, OutlinedInput, MenuItem} from '@mui/material';
+import {useAppDispatch, useAppSelector} from 'store/hooks';
+import {setRealm} from 'store/RealmSlice';
+
+function RealmSelect() {
+  const realm = useAppSelector((state) => state.realm.name);
+  const dispatch = useAppDispatch();
+
+  // realms are hardcoded for now
+  // TODO: should be fetched from keycloak
+  const realms = [
+    {id: 'lha-a', name: 'LHA A', disabled: false},
+    {id: 'lha-b', name: 'LHA B', disabled: true},
+    {id: 'lha-c', name: 'LHA C', disabled: true},
+  ];
+
+  return (
+    <FormControl sx={{m: 2, minWidth: 120}} size='small'>
+      <InputLabel id='login-dialog-realm-select-label'>Realm</InputLabel>
```

Comment:
`Realm` should be internationalized here. I probably also would choose a name a little more descriptive from the user perspective, like "Organization" (German: "Organisation").